src/backend/db/videos.js

```diff
@@ -1,21 +1,200 @@
+import { v4 as uuid } from "uuid";
 /**
  * Videos Database can be added here.
  * You can add videos of your wish with different attributes
  * */
 
 export const videos = [
   {
-    _id: "Wo5dMEP_BbI",
-    title: "Awesome Video about Coding",
+    _id: uuid(),
+    videoId: "hQYRDNl-lGI",
+    title: "The BEST Smartphones of 2021",
     description:
-      "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
-    creator: "Soham Shah",
+      "The Best smartphones of 2021 - including iPhone 13 / 13 Pro, Samsung Galaxy S21, and much more...Get the AI-Powered, Auto-Tracking 4K Webcam -OBSBOT Tiny 4K now with the best deal in all time!",
+    thumbnail:
+      "https://res.cloudinary.com/de5it79dw/image/upload/v1650474888/hq720_heoqot.webp",
+    url: "https://www.youtube.com/watch?v=kwkjQRT42Qk",
+    channelName: "Mrwhosetheboss",
+    channelImage:
+      "https://res.cloudinary.com/de5it79dw/image/upload/v1650475112/channels4_profile_chwng0.jpg",
+    views: 71_174_869,
+    categoryName: "Phone",
   },
   {
-    _id: "F_Riqjdh2oM",
-    title: "Neural Networks from Scratch - P.1 Intro and Neuron Code",
-    creator: "Sentdex",
+    _id: uuid(),
+    videoId: "hQYRDNl-lGI",
+    title: "realme 8 vs Redmi Note 10S Full Comparison",
     description:
-      "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
+      "Doston Is video mein humne kiya hai detailed comparison realme 8 vs Redmi Note 10S ka aur bataaya hai ki kaunsa phone ki taraf aapne dekhna chahiye.",
+    thumbnail:
+      "https://res.cloudinary.com/de5it79dw/image/upload/v1650476803/hq720_1_j2m291.webp",
+    url: "https://www.youtube.com/watch?v=6MtmPoMJQl4",
+    channelName: "Trakin Tech",
+    channelImage:
+      "  https://res.cloudinary.com/de5it79dw/image/upload/v1650476979/channels4_profile_1_up66zk.jpg",
+    views: 71_174_869,
+    categoryName: "Phone",
+  },
+  {
+    _id: uuid(),
+    videoId: "hQYRDNl-lGI",
```

Comment:
You dont need to specify id and videoId separately you can just use videoId in _id as it is already unique